static/app/components/events/interfaces/frame/stacktraceLink.spec.tsx

```diff
@@ -233,22 +233,6 @@ describe('StacktraceLink', function () {
     );
   });
 
-  it('hides stacktrace link if there is no source link for .NET projects', async function () {
```

Comment:
@isabellaenriquez @bruno-garcia I don't think this test is needed anymore since we can derive code mappings for C# (Original PR: #57520).